assets/js/analyzer.js

```diff
@@ -143,7 +143,7 @@ function populateSecondaryAnalyzerList() {
 }
 
 function analyze() {
-    if($('#primaryAnalyzerMode').val() === null) {
+    if($('#primaryAnalyzerMode').val() === null || $('#morphAnalyzerInput').val() === '') {
```

Comment:
I think I might prefer a falsey check, something like `!$('#morphAnalyzerInput').val()`? Also, does this problem also exist with generation? If so, would appreciate an analogous fix in this PR.